atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1654,10 +1669,15 @@ private void waitForCommitToComplete(Iterable<Long> startTimestamps) {
             return;
         }
 
-        WaitForLocksResponse response = timelockService.waitForLocks(
-                WaitForLocksRequest.of(lockDescriptors, LOCK_ACQUISITION_TIMEOUT_MS));
-        Preconditions.checkState(response.wasSuccessful(),
-                "Timed out while waiting for commits to complete");
+        WaitForLocksRequest request = WaitForLocksRequest.of(lockDescriptors, lockAcquireTimeoutMs);
+        WaitForLocksResponse response = timelockService.waitForLocks(request);
+        if (!response.wasSuccessful()) {
+            log.error("Timed out waiting for commits to complete. Timeout was {} ms. First ten locks were {}.",
```

Comment:
We decided that we still need them on the client side, since large internal product does not use an sls-spec logger